src/main/java/net/sf/jsqlparser/util/deparser/UpdateDeParser.java

```diff
@@ -90,7 +90,11 @@ public void deParse(Update update) {
 
         if (update.getWhere() != null) {
             buffer.append(" WHERE ");
-            update.getWhere().accept(expressionVisitor);
+            int len = buffer.length();
```

Comment:
Please remove those specific parts from the PR as it does not fit into the generic Deparser.